dmri_seg.yml

```diff
@@ -0,0 +1,14 @@
+name: dmri_seg
```

Comment:
Ryan, I said overwrite `environment_gpu.yml` file.

Reply:
Oh okay, that's right, sorry about that.